tests/__init__.py

```diff
@@ -21,5 +21,5 @@
 from .jobs import *
 from .impersonation import *
 from .models import *
-from .operators import *
+# from .operators import *
```

Comment:
can you remove it

Reply:
shit, I forgot about that. No. I have to have it. I just can't run tests locally unless I comment it out